Automattic/jetpack
# Dashboard: add highlight of Jetpack blocks to the settings page

#### Changes proposed in this Pull Request:

﻿Related: #23468

Let's promote that Jetpack includes blocks in the Jetpack settings, in the Jetpack dashboard.

<img width="1087" alt="image" src="https://user-images.githubusercontent.com/426388/158655527-81b89c5e-98dc-4ca5-aba9-734cb94a2709.png">


Later on, we can add a toggle here that would act as a main toggle from this:
https://github.com/Automattic/jetpack/blob/a09c00ea3c1531eb15d5d8f58bc685dd7d916799/projects/plugins/jetpack/class.jetpack-gutenberg.php#L451-L474

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Go to Jetpack > Settings > Writing
* You should see a card promoting Jetpack blocks, with a link to the block editor. Give it a try.
* Go to Plugins > Add New, and install / activate the Classic Editor plugin.
* Go back to Jetpack > Settings > Writing, the blocks card should be gone.
